GenderPayGap.UnitTests/GenderPayGap.WebUI.Tests/Controllers/Administration/Scope/ScopeControllerTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using GenderPayGap.Core;
+using GenderPayGap.Core.Classes;
+using GenderPayGap.Database;
+using GenderPayGap.Extensions;
+using GenderPayGap.WebUI.Models.Admin;
+using GenderPayGap.WebUI.Tests.Builders;
+using GenderPayGap.WebUI.Tests.TestHelpers;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Primitives;
+using NUnit.Framework;
+
+namespace GenderPayGap.WebUI.Tests.Controllers.Scope
+{
+    [TestFixture]
+    [SetCulture("en-GB")]
+    public class ScopeControllerTests
+    {
+
+        [Test]
+        [Description("POST: Existing scopes are retired when a new scope is added")]
+        public void POST_Existing_Scopes_Are_Retired_When_New_Scope_Is_Added()
+        {
+            // Arrange
+            User user = new UserBuilder();
+
+            Organisation organisation = new OrganisationBuilder().WithSectorType(SectorTypes.Private);
+
+            OrganisationScope organisationScope2018 = new OrganisationScopeBuilder().WithOrganisation(organisation)
+                .WithReportingYear(2018);
+            
+            OrganisationScope organisationScope2017 = new OrganisationScopeBuilder().WithOrganisation(organisation)
+                .WithReportingYear(2017);
+
+            var requestFormValues = new Dictionary<string, StringValues>();
+            requestFormValues.Add("GovUk_Radio_NewScopeStatus", "OutOfScope");
+            requestFormValues.Add("GovUk_Text_Reason", "A reason");
+
+            object[] dbObjects = {user, organisation, organisationScope2017, organisationScope2018};
+
+            var controller = NewUiTestHelper.GetController<WebUI.Controllers.AdminOrganisationScopeController>(requestFormValues: requestFormValues, dbObjects: dbObjects);

```

Comment:
* I've pushed a change to use a new `ControllerBuilder` that (I think) makes this a bit neater
* Are you deliberately  testing the **AdminOrganisationScopeController**?
Your previous work was on the **ScopeController**